cfpb/capital-framework-sandbox
# Updated the sandbox for latest icon changes in prerelease-ver-4

This PR updates the Sandbox to match the latest icon changes in prerelease-ver-4. It does not include the removed code in [PR 418](https://github.com/cfpb/capital-framework/pull/418).

## Additions

- Added cf-icons to the list of npm sym-linked modules

## Changes

- Updated the cf-icons examples to include the pseudo-elem modifier.

## Testing

- Checkout the latest commit to `prerelease-ver-4` and run `npm run cf-link`
- In this repo run `npm run cf-link && npm run build && npm start`
- Navigate to `http://localhost:3000/components/cf-icons/` Everything should look the same.

## Review

- @Scotchester 
- @contolini 

[Preview this PR without the whitespace changes](?w=0)

## Todos

- Remove the duplicated icon code from links and buttons

## Checklist
